src/geometry/euclidean/circumcenter.lean

```diff
@@ -335,6 +335,79 @@ begin
   exact h.2
 end
 
+/-- The circumradius is non-negative. -/
+lemma circumradius_nonneg {n : ℕ} (s : simplex ℝ P n) : 0 ≤ s.circumradius :=
+s.dist_circumcenter_eq_circumradius 0 ▸ dist_nonneg
+
+/-- The circumradius of a simplex with at least two points is
+positive. -/
+lemma circumradius_pos {n : ℕ} (s : simplex ℝ P (n + 1)) : 0 < s.circumradius :=
+begin
+  refine lt_of_le_of_ne s.circumradius_nonneg _,
+  intro h,
+  have hr := s.dist_circumcenter_eq_circumradius,
+  simp_rw [←h, dist_eq_zero] at hr,
+  have h01 := (injective_of_affine_independent s.independent).ne
+    (dec_trivial : (0 : fin (n + 2)) ≠ 1),
+  simpa [hr] using h01
+end
+
+/-- The circumcenter of a 0-simplex equals its unique point. -/
+lemma circumcenter_eq_point (s : simplex ℝ P 0) (i : fin 1) :
+  s.circumcenter = s.points i :=
```

Comment:
If you swap the two sides, this could be a simp-lemma.

Reply:
I don't think it should be a simp lemma. The simplest form is probably `s.points 0` (but this lemma isn't requiring that the index you're given is exactly `(0 : fin 1)`, although of course it must be equal).

Reply:
Ok... if you are interested in building a simp-set, then you could add a lemma that simps the circumcenter into `s.points 0`. Otherwise, just leave it like this.

Reply:
I'm not actually sure if `simp` would ever be useful here or not, or indeed whether 0-simplices would ever appear in practice. It's `circumcenter_eq_centroid` for a 1-simplex that I'm actually using, the 0-simplex case is just included here for completeness.